policy/modules/system/systemd.te

```diff
@@ -1451,9 +1466,8 @@ allow systemd_sessions_t systemd_sessions_runtime_t:file manage_file_perms;
 files_runtime_filetrans(systemd_sessions_t, systemd_sessions_runtime_t, file)
 
 fs_getattr_all_fs(systemd_sessions_t)
+fs_getattr_cgroup(systemd_sessions_t)
 fs_search_cgroup_dirs(systemd_sessions_t)
-fs_search_tmpfs(systemd_sessions_t)
-fs_search_ramfs(systemd_sessions_t)
```

Comment:
I don't understand the deletions. There's the getattr all, but no search all. I think you meant to remove `fs_getattr_cgroup(systemd_sessions_t)` from your patch?

Reply:
Ah yes, sorry for the confusion. That is exactly what I wanted to do.